### ISSUE TITLE
White Box

### ISSUE THREAD

**Issue by Wallvon** [sytespace/supportapp#1]
There is a white box underneath the form.
![image](https://user-images.githubusercontent.com/42699143/61174140-88cb0b00-a59c-11e9-8746-f0842f672022.png)

**Comment by PiggyPlex** [comment]
Empty iframe most likely

**Comment by PiggyPlex** [comment]
Yeah, it appears so.
Take note of the second iframe. Perhaps those properties were meant for the first one, but by accident they made a new one?
```html
<iframe src="https://docs.google.com/forms/d/e/1FAIpQLSebKPoSOHK0UXqLmRnOgQhm1Nkdb-LAluszijxjVT460aTClw/viewform?embedded=true" width="1000" height="1626" frameborder="0" marginheight="0" marginwidth="0">Loading...</iframe>
--
  | <iframe align="left\|right\|middle\|top\|bottom">
```

**Comment by MM-coder** [comment]
Has been fixed in  #007d7f2
